<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameDetailsActivity</class>
 <widget class="QWidget" name="GameDetailsActivity">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="mp_layout">
     <item>
      <widget class="QPushButton" name="mp_btn_close">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
       <property name="icon">
        <iconset resource="../Resources/Resources.qrc">
         <normaloff>:/images/back</normaloff>:/images/back</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="OplPcTools::UI::ClickableLabel" name="mp_label_title">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="mp_btn_download_all_art">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="toolTip">
        <string>Download All Pictures...</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../Resources/Resources.qrc">
         <normaloff>:/images/download</normaloff>:/images/download</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="mp_btn_rename_game">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="toolTip">
        <string>Rename...</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../Resources/Resources.qrc">
         <normaloff>:/images/rename</normaloff>:/images/rename</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="mp_tabs">
     <property name="tabPosition">
      <enum>QTabWidget::South</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="mp_tab_arts">
      <attribute name="title">
       <string>Pictures</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
     <widget class="QWidget" name="mp_tab_config">
      <attribute name="title">
       <string>Configuration</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OplPcTools::UI::ClickableLabel</class>
   <extends>QLabel</extends>
   <header location="global">OplPcTools/UI/ClickableLabel.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mp_btn_download_all_art</tabstop>
  <tabstop>mp_btn_rename_game</tabstop>
  <tabstop>mp_tabs</tabstop>
  <tabstop>mp_btn_close</tabstop>
 </tabstops>
 <resources>
  <include location="../Resources/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
